guides/common/modules/proc_upgrading-a-package.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Upgrading_a_package_{context}"]
+= Upgrading a package
+
+Use the following procedure to view the installed packages of a host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the name of the host that contains the packages you want to remove.
+. On the new Hosts page, select the *Content* tab.
+. On the *Content* page select the *Packages* tab.
+. On the *Packages* page the *Status* column displays whether the package is upgradable or Up to date.
+You cannot update an up-to-date package.
+. From the list of packages, choose the package you want to upgrade and click the three dots icon at the end of the line.
+. Choose the *Upgrade via Remote Execution* option if UNKNOWN or *Upgrade via customized remote execution* if UNKNOWN.
```

Comment:
the same as above applies, I suppose here we no longer offer a katello agent option, since we strongly prefer to use REX